src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -264,14 +273,34 @@ const ComposeForm = props => {
 
   useEffect(
     () => {
-      if (
-        messageInvalid ||
-        (isSignatureRequired && (signatureInvalid || checkboxInvalid))
-      ) {
-        focusOnErrorField();
-      }
+      const handleEffect = async () => {
+        const selectedTG = recipientsList.find(
+          recipient => recipient.triageTeamId === +selectedRecipientId,
+        );
+        if (
+          alertDisplayed &&
+          !alertHasFocused &&
+          alertRef.current &&
+          selectedTG
+        ) {
+          await focusElement(selectedTG?.name);
+          focusElement(alertRef.current);
```

Comment:
interesting, what are you trying to achieve with calling one `focusElement` right after another?

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/31543#discussion_r1744542971